docs/pages/eas-update/rollouts.mdx

```diff
@@ -3,6 +3,25 @@ title: Rollouts
 hideTOC: true
 ---
 
-EAS Update does not support percentage-based rollouts. With that said, we built EAS Update with percentage-based rollouts in mind. We plan to work on rollouts in 2023. With this feature, you'll be able to incrementally roll out one update over another to a percentage of your end-users.
+Rollouts allow us to deploy a new branch to a specific channel incrementally. With rollouts, we can roll out Branch B over Branch A by percentage. This is useful when we want to test a new feature to minimize the risk of introducing bugs or other issues to our production environment.
+
+To initiate a rollout, we can use the following command:
+
+```bash
+eas channel:rollout [channel]
+```
```

Comment:
```suggestion
<Terminal cmd={["$ eas channel:rollout [channel]"]} />
```

Let's use `<Terminal>` component here for shell scripts: https://github.com/expo/expo/tree/main/docs#use-terminal-component-for-shell-commands-snippets

Reply:
I removed the necessity for the [channel] arg (it prompts for it if not supplied)